internal/cli/atlas/deployments/setup.go

```diff
@@ -177,15 +177,18 @@ func (opts *SetupOpts) configureMongod(ctx context.Context, keyFileContents stri
 			Cmd: "/bin/sh",
 			Args: []string{
 				"-c",
-				`echo $KEYFILECONTENTS > $KEYFILE && \
-                   chmod 400 $KEYFILE && \
-                   python3 /usr/local/bin/docker-entrypoint.py \
-                                          --transitionToAuth \
-                                          --dbpath $DBPATH \
-                                          --keyFile $KEYFILE \
-                                          --replSet $REPLSETNAME \
-                                          --setParameter "mongotHost=$MONGOTHOST" \
-                                          --setParameter "searchIndexManagementHostAndPort=$MONGOTHOST"`,
+				`if [ ! -f $KEYFILE ]
+                 then
+                     echo $KEYFILECONTENTS > $KEYFILE
+                     chmod 400 $KEYFILE
+                 fi
+                 python3 /usr/local/bin/docker-entrypoint.py \
```

Comment:
[q] do we need to capture the python3 dependency as well somewhere or do all OS come with it by default?

Reply:
ah nvm me :D